### PR TITLE
adding test for kubernetescluster restart node command and update az-…

### PR DESCRIPTION
**Pull request by MikePham0630**
…cli-extension test for v20230701

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/networkcloud/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+1.0.1
```
Comment (kairu-ms) [inline src/networkcloud/HISTORY.rst:6]: Please use version `1.1.0` as this is not a patch change.

File: src/networkcloud/setup.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
 
 
 # HISTORY.rst entry.
-VERSION = '1.0.0'
+VERSION = '1.0.1'
```
Comment (kairu-ms) [inline src/networkcloud/setup.py:13]:
```suggestion
VERSION = '1.1.0'
```

**Comment by priyamshet** [comment]
Please use https://github.com/Azure/azure-cli-extensions/pull/6731  for the latest changes.

**Pull request closed without merging**